src/ontology/foodon-edit.owl

```diff
@@ -18825,6 +18826,16 @@ AnnotationAssertion(dc:date obo:FOODON_00004130 "2021-12-14T21:47:32Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00004130 "food mixture"@en)
 SubClassOf(obo:FOODON_00004130 obo:NCIT_C45305)
 
+# Class: obo:FOODON_00004146 (beef formula)
```

Comment:
The definition of this needs to be fixed.  If you read the definition alone would you be able to guess what the food item was?  Is it a diet?  Is it a marketing campaign?  Also should this be in foodon_product_import.owl ?

Reply:
Hmm, infact now thinking if we should have this term in FoodOn? Because formula is a subjective recipe. Different brands may use different ratio of beef and supplements. Ingredients may vary as well.

Reply:
Yes that works - though foodon is using "defining" ingredient probably rather than "prime" ingredient.

Reply:
okay great, thanks!

Reply:
> The definition of this needs to be fixed. If you read the definition alone would you be able to guess what the food item was? Is it a diet? Is it a marketing campaign? Also should this be in foodon_product_import.owl ?

Yes, it should ideally be in foodon_product_import.owl, but because I placed it under 'animal feed supplement' earlier which was only in foodon-edit.owl. But anyways I've changed the parent class to 'food product for animal', as it is also used as a complete meal too other than supplement. 
Also,I have moved the term to foodon_product_import.owl.